go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/confio/ics23/go v0.9.0
 	github.com/cosmos/admin-module v0.0.0-00010101000000-000000000000
 	github.com/cosmos/cosmos-proto v1.0.0-beta.3
-	github.com/cosmos/cosmos-sdk v0.45.15
+	github.com/cosmos/cosmos-sdk v0.46.11
```

Comment:
We can't do such an upgrade, please leave it as it is. We are going to upgrade directly to SDK 47 soon